tests/unit/controlgroup/controlgroup_methods.js

```diff
@@ -0,0 +1,130 @@
+module( "Controlgroup: methods" );
+
+test( "destroy", function() {
+	expect( 1 );
+	domEqual( ".controlgroup", function() {
+		$( ".controlgroup" ).controlgroup().controlgroup( "destroy" );
+	});
+});
+test( "disable", function() {
+	var tests = 1,
+		element = $( ".controlgroup" ).controlgroup().controlgroup( "disable" );
+	strictEqual( element.hasClass( "ui-state-disabled" ), false,
+		"ui-state-disabled is not present on widget after disabling" );
+	$.each( $.ui.controlgroup.prototype.options.items, function( widget, selector ){
```

Comment:
This is duplicating too much logic of the widget itself. Since this produces only 6 assertions (7 in total, one is above), we could as well just list them out. Or extract into a helper and reuse it in the enable test below, passing the expected boolean as a parameter. Could also merge the two tests into one.